src/index.js

```diff
@@ -3,4 +3,20 @@ import './assets/github.svg';
 import './assets/linkedin.svg';
 import './assets/medal.svg';
 import 'bootstrap';
-// import 'bootstrap/dist/css/bootstrap.min.css';
+import displayData from './modules/displayData.js';
+import postScores from './modules/postScores.js';
+
+const formBtn = document.querySelector('.formBtn');
+
+formBtn.addEventListener('click', (ev) => {
+  ev.preventDefault();
+
+  const form = document.getElementById('form');
+  const user = form.user.value;
+  const score = form.score.value;
+  postScores(user, score);
+  form.reset();
+});
+
+const refreshBtn = document.querySelector('#refreshBtn');
+refreshBtn.onclick = displayData;
```

Comment:
- [ ] Excellent work on achieving the API integration milestone! However, during my review, I observed that the scores are not being fetched on page load; they are only retrieved when the user clicks the `refresh button.` This behavior is not intended, and we need the scores to be fetched automatically without requiring the user to click anything.

   - To resolve this issue, I suggest adding a line of code to make use of the `window.onload` event to trigger the `displayData` function when the page finishes loading. By doing so, the scores will be fetched and displayed as soon as the page is ready, providing the desired behavior.